llvm/utils/TableGen/VTEmitter.cpp

```diff
@@ -130,6 +130,61 @@ void VTEmitter::run(raw_ostream &OS) {
     // clang-format on
   }
   OS << "#endif\n\n";
+
+  OS << "#ifdef GET_VT_EVT\n";
+  for (const auto *VT : VTsByNumber) {
+    if (!VT)
+      continue;
+    bool IsInteger = VT->getValueAsBit("isInteger");
```

Comment:
Split this whole thing out into a VT to StringRef name helper function?

Reply:
Extract this into VTtoGetLLVMTyString now :)